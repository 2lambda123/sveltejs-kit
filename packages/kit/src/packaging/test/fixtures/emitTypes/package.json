{
	"name": "emit-types",
	"version": "1.0.0",
	"description": "emitTypes-disabled",
	"type": "module"
}
