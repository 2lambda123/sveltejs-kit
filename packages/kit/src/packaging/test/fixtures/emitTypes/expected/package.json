{
	"name": "emit-types",
	"version": "1.0.0",
	"description": "emitTypes-disabled",
	"type": "module",
	"exports": {
		"./package.json": "./package.json",
		"./Test.svelte": "./Test.svelte",
		"./Test2.svelte": "./Test2.svelte",
		".": "./index.js"
	}
}
